src/components/common/OfficeAddresses.vue

```diff
@@ -130,7 +132,9 @@
               <div class="meta-container">
                 <label>{{ showAddressForm ? "Mailing Address" : "" }}</label>
                 <div class="meta-container__inner">
-                  <label v-if="!isSame(recDeliveryAddress, recMailingAddress, 'actions') && !showAddressForm">
+                  <label v-if="!showAddressForm &&
```

Comment:
In this "and" clause... same comment as for "or" clause.

(Because the execution may not have to check the remaining conditions.